src/app/modules/main/wallet_section/collectibles/current_collectible/view.nim

```diff
@@ -197,6 +215,8 @@ QtObject:
     self.tokenIdChanged()
     self.descriptionChanged()
     self.backgroundColorChanged()
+    self.mediaUrlChanged()
```

Comment:
@dlipicar looking at this `setData` proc now, I am wondering why don't we use the same signal like `proc currentCollectibleChanged(self: View) {.signal.}` and set it as `notify` signal for all exposed properties from here?

That approach may have less performance impact than emitting bunch of signals at the same time or emitting signals in a row as soon as you change a value for a property, one by one. Doing like it's now causes reevaluations on the qml side and also triggering some bindings but that's not necessary since all those props are changed at once.

Let's change that, wdyt?

Reply:
yeah, sounds much cleaner, thanks for the suggestion!